{"uid":"16cf62d463b887e9","name":"Drag and drop card from one list to another","fullName":"tests.regression_ui.test_ui_regression.TestUIRegression#test_drag_n_drop_card","historyId":"58c27d4ea8138de64fd4e105da17ea92","time":{"start":1734373345102,"stop":1734373379635,"duration":34533},"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting","statusTrace":"self = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7f7cec0aa510>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"9eb150a427670ed7d3d7c86e688c6caf\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC001\n    @allure.title('Login Functionality')\n    def test_login(self, driver):\n        with allure.step('Navigate to the Trello login page'):\n            driver.get(self.SCC.URL_LOGIN)\n            time.sleep(3)\n        with allure.step('Enter valid credentials (email, password, and 6-digits verification code)'):\n            try:\n                el_user_name = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.USER_NAME_FIELD)\n                ))\n                el_user_name.click()\n                el_user_name.send_keys(USER_EMAIL)\n    \n                driver.find_element(By.XPATH, self.SCC.LoginPage.CONTINUE_BUTTON).click()\n                el_password = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.PASSWORD_FIELD)\n                ))\n                el_password.click()\n                el_password.send_keys(USER_PASSWORD)\n    \n                driver.find_element(By.XPATH, self.SCC.LoginPage.LOGIN_BUTTON).click()\n                time.sleep(3)\n                el_six_otp_code = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.SIX_OTP_CODE_FIELD)\n                ))\n                el_six_otp_code.click()\n                el_six_otp_code.send_keys(self.otp_auth())\n                time.sleep(10)\n>               WebDriverWait(driver, 10).until(EC.url_matches(self.SCC.HOME_URL))\n\ntests/regression_ui/test_ui_regression.py:84: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"9eb150a427670ed7d3d7c86e688c6caf\")>\nmethod = <function url_matches.<locals>._predicate at 0x7f7cec03b1a0>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7f7cec0aa510>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"9eb150a427670ed7d3d7c86e688c6caf\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC005\n    @allure.title('Drag and drop card from one list to another')\n    def test_drag_n_drop_card(self, driver):\n        time.sleep(7)\n        with allure.step('Log in to Trello account'):\n>           self.test_login(driver)\n\ntests/regression_ui/test_ui_regression.py:166: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/regression_ui/test_ui_regression.py:88: in test_login\n    log.info(\"An error occurred: \", str(e))\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:2216: in info\n    root.info(msg, *args, **kwargs)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1539: in info\n    self._log(INFO, msg, args, **kwargs)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1684: in _log\n    self.handle(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1700: in handle\n    self.callHandlers(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1762: in callHandlers\n    hdlr.handle(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1028: in handle\n    self.emit(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py:386: in emit\n    super().emit(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1168: in emit\n    self.handleError(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1160: in emit\n    msg = self.format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:999: in format\n    return fmt.format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py:139: in format\n    return super().format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:703: in format\n    record.message = record.getMessage()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <LogRecord: root, 20, /home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py, 88, \"An error occurred: \">\n\n    def getMessage(self):\n        \"\"\"\n        Return the message for this LogRecord.\n    \n        Return the message for this LogRecord after merging any user-supplied\n        arguments with the message.\n        \"\"\"\n        msg = str(self.msg)\n        if self.args:\n>           msg = msg % self.args\nE           TypeError: not all arguments converted during string formatting\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:392: TypeError","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"driver","time":{"start":1734373344841,"stop":1734373345101,"duration":260},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"setup","time":{"start":1734373345101,"stop":1734373345102,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting","statusTrace":"self = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7f7cec0aa510>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"9eb150a427670ed7d3d7c86e688c6caf\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC001\n    @allure.title('Login Functionality')\n    def test_login(self, driver):\n        with allure.step('Navigate to the Trello login page'):\n            driver.get(self.SCC.URL_LOGIN)\n            time.sleep(3)\n        with allure.step('Enter valid credentials (email, password, and 6-digits verification code)'):\n            try:\n                el_user_name = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.USER_NAME_FIELD)\n                ))\n                el_user_name.click()\n                el_user_name.send_keys(USER_EMAIL)\n    \n                driver.find_element(By.XPATH, self.SCC.LoginPage.CONTINUE_BUTTON).click()\n                el_password = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.PASSWORD_FIELD)\n                ))\n                el_password.click()\n                el_password.send_keys(USER_PASSWORD)\n    \n                driver.find_element(By.XPATH, self.SCC.LoginPage.LOGIN_BUTTON).click()\n                time.sleep(3)\n                el_six_otp_code = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.SIX_OTP_CODE_FIELD)\n                ))\n                el_six_otp_code.click()\n                el_six_otp_code.send_keys(self.otp_auth())\n                time.sleep(10)\n>               WebDriverWait(driver, 10).until(EC.url_matches(self.SCC.HOME_URL))\n\ntests/regression_ui/test_ui_regression.py:84: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"9eb150a427670ed7d3d7c86e688c6caf\")>\nmethod = <function url_matches.<locals>._predicate at 0x7f7cec03b1a0>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7f7cec0aa510>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"9eb150a427670ed7d3d7c86e688c6caf\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC005\n    @allure.title('Drag and drop card from one list to another')\n    def test_drag_n_drop_card(self, driver):\n        time.sleep(7)\n        with allure.step('Log in to Trello account'):\n>           self.test_login(driver)\n\ntests/regression_ui/test_ui_regression.py:166: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/regression_ui/test_ui_regression.py:88: in test_login\n    log.info(\"An error occurred: \", str(e))\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:2216: in info\n    root.info(msg, *args, **kwargs)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1539: in info\n    self._log(INFO, msg, args, **kwargs)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1684: in _log\n    self.handle(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1700: in handle\n    self.callHandlers(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1762: in callHandlers\n    hdlr.handle(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1028: in handle\n    self.emit(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py:386: in emit\n    super().emit(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1168: in emit\n    self.handleError(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1160: in emit\n    msg = self.format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:999: in format\n    return fmt.format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py:139: in format\n    return super().format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:703: in format\n    record.message = record.getMessage()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <LogRecord: root, 20, /home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py, 88, \"An error occurred: \">\n\n    def getMessage(self):\n        \"\"\"\n        Return the message for this LogRecord.\n    \n        Return the message for this LogRecord after merging any user-supplied\n        arguments with the message.\n        \"\"\"\n        msg = str(self.msg)\n        if self.args:\n>           msg = msg % self.args\nE           TypeError: not all arguments converted during string formatting\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:392: TypeError","steps":[{"name":"Log in to Trello account","time":{"start":1734373352102,"stop":1734373379634,"duration":27532},"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting\n","statusTrace":"  File \"/home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py\", line 166, in test_drag_n_drop_card\n    self.test_login(driver)\n  File \"/home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py\", line 88, in test_login\n    log.info(\"An error occurred: \", str(e))\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 2216, in info\n    root.info(msg, *args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1539, in info\n    self._log(INFO, msg, args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1684, in _log\n    self.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1700, in handle\n    self.callHandlers(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1762, in callHandlers\n    hdlr.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1028, in handle\n    self.emit(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py\", line 386, in emit\n    super().emit(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1168, in emit\n    self.handleError(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1160, in emit\n    msg = self.format(record)\n          ^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 999, in format\n    return fmt.format(record)\n           ^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py\", line 139, in format\n    return super().format(record)\n           ^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 703, in format\n    record.message = record.getMessage()\n                     ^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 392, in getMessage\n    msg = msg % self.args\n          ~~~~^~~~~~~~~~~\n","steps":[{"name":"Navigate to the Trello login page","time":{"start":1734373352102,"stop":1734373355398,"duration":3296},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Enter valid credentials (email, password, and 6-digits verification code)","time":{"start":1734373355398,"stop":1734373379633,"duration":24235},"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting\n","statusTrace":"  File \"/home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py\", line 88, in test_login\n    log.info(\"An error occurred: \", str(e))\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 2216, in info\n    root.info(msg, *args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1539, in info\n    self._log(INFO, msg, args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1684, in _log\n    self.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1700, in handle\n    self.callHandlers(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1762, in callHandlers\n    hdlr.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1028, in handle\n    self.emit(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py\", line 386, in emit\n    super().emit(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1168, in emit\n    self.handleError(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1160, in emit\n    msg = self.format(record)\n          ^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 999, in format\n    return fmt.format(record)\n           ^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py\", line 139, in format\n    return super().format(record)\n           ^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 703, in format\n    record.message = record.getMessage()\n                     ^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 392, in getMessage\n    msg = msg % self.args\n          ~~~~^~~~~~~~~~~\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":2,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"e1e4ea8f9c3d2177","name":"log","source":"e1e4ea8f9c3d2177.txt","type":"text/plain","size":63294}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":3,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"driver::0","time":{"start":1734373379762,"stop":1734373379835,"duration":73},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"TC005"},{"name":"tag","value":"TC000"},{"name":"parentSuite","value":"tests.regression_ui"},{"name":"suite","value":"test_ui_regression"},{"name":"subSuite","value":"TestUIRegression"},{"name":"host","value":"fv-az1922-539"},{"name":"thread","value":"3617-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.regression_ui.test_ui_regression"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":7,"skipped":0,"passed":23,"unknown":0,"total":30},"items":[{"uid":"caa2924ed47e2b46","status":"passed","time":{"start":1734311643843,"stop":1734311674223,"duration":30380}},{"uid":"5a118b2f2f42ca95","status":"passed","time":{"start":1734286799085,"stop":1734286829502,"duration":30417}},{"uid":"b92fdddfc7f7d936","status":"passed","time":{"start":1734225358079,"stop":1734225388370,"duration":30291}},{"uid":"d9ac6093d7298c69","status":"passed","time":{"start":1734200409364,"stop":1734200440133,"duration":30769}},{"uid":"81137cc199dbf242","status":"passed","time":{"start":1734185909733,"stop":1734185940327,"duration":30594}},{"uid":"3b43405534010ba7","status":"passed","time":{"start":1734124047566,"stop":1734124077912,"duration":30346}},{"uid":"d2adfdd9feba6fa4","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x559859e8f31a <unknown>\n#1 0x5598599a56e0 <unknown>\n#2 0x5598599f43e6 <unknown>\n#3 0x5598599f4681 <unknown>\n#4 0x559859a39b04 <unknown>\n#5 0x559859a1848d <unknown>\n#6 0x559859a36ed7 <unknown>\n#7 0x559859a18203 <unknown>\n#8 0x5598599e6cc0 <unknown>\n#9 0x5598599e7c9e <unknown>\n#10 0x559859e5cd0b <unknown>\n#11 0x559859e60c92 <unknown>\n#12 0x559859e49b3c <unknown>\n#13 0x559859e61807 <unknown>\n#14 0x559859e2f0df <unknown>\n#15 0x559859e7e578 <unknown>\n#16 0x559859e7e740 <unknown>\n#17 0x559859e8e196 <unknown>\n#18 0x7feaee094ac3 <unknown>","time":{"start":1734113697163,"stop":1734113749831,"duration":52668}},{"uid":"f7a1644cb9436635","status":"passed","time":{"start":1734050860376,"stop":1734050890660,"duration":30284}},{"uid":"2ebf221c7a6de23f","status":"passed","time":{"start":1734027457230,"stop":1734027487602,"duration":30372}},{"uid":"8f0c4b250af0056c","status":"passed","time":{"start":1733973827957,"stop":1733973858243,"duration":30286}},{"uid":"2969764b1dedd2ea","status":"passed","time":{"start":1733972693839,"stop":1733972724206,"duration":30367}},{"uid":"9d1a355e4661da26","status":"passed","time":{"start":1733971728210,"stop":1733971758553,"duration":30343}},{"uid":"6631560d10ac6ea8","status":"passed","time":{"start":1733970474000,"stop":1733970504493,"duration":30493}},{"uid":"b53fcb4772f3314a","status":"passed","time":{"start":1733963373210,"stop":1733963404021,"duration":30811}},{"uid":"c67eb4dc3c2a9aae","status":"passed","time":{"start":1733932645122,"stop":1733932675979,"duration":30857}},{"uid":"8818901738086322","status":"passed","time":{"start":1733891486880,"stop":1733891517227,"duration":30347}},{"uid":"52faaabaa5c8223","status":"passed","time":{"start":1733891042705,"stop":1733891072993,"duration":30288}},{"uid":"bea2e597fe4c4073","status":"broken","statusDetails":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//*[contains(text(), 'new_card')]\"}\n  (Session info: chrome=131.0.6778.139); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55623297d31a <unknown>\n#1 0x5562324936e0 <unknown>\n#2 0x5562324e23e6 <unknown>\n#3 0x5562324e2681 <unknown>\n#4 0x556232527b04 <unknown>\n#5 0x55623250648d <unknown>\n#6 0x556232524ed7 <unknown>\n#7 0x556232506203 <unknown>\n#8 0x5562324d4cc0 <unknown>\n#9 0x5562324d5c9e <unknown>\n#10 0x55623294ad0b <unknown>\n#11 0x55623294ec92 <unknown>\n#12 0x556232937b3c <unknown>\n#13 0x55623294f807 <unknown>\n#14 0x55623291d0df <unknown>\n#15 0x55623296c578 <unknown>\n#16 0x55623296c740 <unknown>\n#17 0x55623297c196 <unknown>\n#18 0x7fc2c4a94ac3 <unknown>","time":{"start":1733890504137,"stop":1733890533844,"duration":29707}},{"uid":"73438c47b7e95492","status":"broken","statusDetails":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//*[contains(text(), 'new_card')]\"}\n  (Session info: chrome=131.0.6778.139); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x564ef32cc31a <unknown>\n#1 0x564ef2de26e0 <unknown>\n#2 0x564ef2e313e6 <unknown>\n#3 0x564ef2e31681 <unknown>\n#4 0x564ef2e76b04 <unknown>\n#5 0x564ef2e5548d <unknown>\n#6 0x564ef2e73ed7 <unknown>\n#7 0x564ef2e55203 <unknown>\n#8 0x564ef2e23cc0 <unknown>\n#9 0x564ef2e24c9e <unknown>\n#10 0x564ef3299d0b <unknown>\n#11 0x564ef329dc92 <unknown>\n#12 0x564ef3286b3c <unknown>\n#13 0x564ef329e807 <unknown>\n#14 0x564ef326c0df <unknown>\n#15 0x564ef32bb578 <unknown>\n#16 0x564ef32bb740 <unknown>\n#17 0x564ef32cb196 <unknown>\n#18 0x7fd438c94ac3 <unknown>","time":{"start":1733889882357,"stop":1733889912076,"duration":29719}},{"uid":"f71a3d68a7173f38","status":"broken","statusDetails":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//*[contains(text(), 'new_card')]\"}\n  (Session info: chrome=131.0.6778.139); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55a402bbd31a <unknown>\n#1 0x55a4026d36e0 <unknown>\n#2 0x55a4027223e6 <unknown>\n#3 0x55a402722681 <unknown>\n#4 0x55a402767b04 <unknown>\n#5 0x55a40274648d <unknown>\n#6 0x55a402764ed7 <unknown>\n#7 0x55a402746203 <unknown>\n#8 0x55a402714cc0 <unknown>\n#9 0x55a402715c9e <unknown>\n#10 0x55a402b8ad0b <unknown>\n#11 0x55a402b8ec92 <unknown>\n#12 0x55a402b77b3c <unknown>\n#13 0x55a402b8f807 <unknown>\n#14 0x55a402b5d0df <unknown>\n#15 0x55a402bac578 <unknown>\n#16 0x55a402bac740 <unknown>\n#17 0x55a402bbc196 <unknown>\n#18 0x7f9065294ac3 <unknown>","time":{"start":1733888739225,"stop":1733888768930,"duration":29705}}]},"tags":["TC005","TC000"]},"source":"16cf62d463b887e9.json","parameterValues":[]}